packages/cryptoplease_ui/lib/src/info_widget.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:cryptoplease_ui/cryptoplease_ui.dart';
+import 'package:flutter/material.dart';
+
+class CpInfoWidget extends StatelessWidget {
```

Comment:
For all the widgets in `cryptoplease_ui` we add a corresponding story in the storybook. Please, add the story for this widget.